packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Overview of the mobile Identity Engine SDK
+---
+
+<div class="oie-embedded-sdk">
+
+Design the implementation of sign-in for your mobile app by understanding the
+representation of the flow, and with examples of the common parts.
+
+<ApiLifecycle access="ie" /><br>
+
+## Introduction
+
+Integrate the Okta sign-in flow into your app in one of two ways: redirect the request to
+Okta which handles the flow in it's own window, or handle each step in the flow by
+building the appropriate user interface elements, capturing the information, and updating
+Okta.
+
+This guide is an overview of building your own user interface. For information on the redirect model, see <StackSnippet snippet="redirectquickstart" inline />.
+
+## Sign-in flow
+
+Okta supports many ways of authenticating the identity of a user during sign-in. An Okta org administrator creates _policies_, different mixes of authentication methods (also
+called factors), and assigns them to apps, people, groups, and more. Policies also
+configure how many authentication methods are required (multifactor), and which methods
+are optional. This results in a large number of possible combinations for the different
+factors and steps to complete the sign-in flow.
+
+The Android and iOS Identity Engine SDKs represent the sign-in flow as a state machine.
+You initialize the machine with the details of your Okta Application Integration, request
+the initial step in the flow, and cycle through responding to steps until either the user
+is signed in, cancels, or an error occurs.
+
+<div class="common-image-format">
+
+![A diagram showing the sign-in flow.](/img/mobile-sdk/mobile-idx-basic-flow.png "A diagram that shows the sign-in flow.")
+
+</div>
+
+Each sign-in step may include one or more possible user actions, such as:
+- Choosing an authenticator, such as Okta Verify or WebAuthN.
+- Entering a One Time Passcode (OTP).
+- Cancelling the sign-in.
+
+## Sign-in objects
+
+Each step in the flow is represented by a _Response_ which contains the information used
```

Comment:
I'm using emphasis because this is a technical term that the developer may never have seen and it's important for them to remember the definition. It's capitalized because both the iOS and Android IDX SDKs include an object with that name.

I though of using code voice, would that be more appropriate?

Reply:
Ah it's an object, my bad. Leave it capitalized. I struggle with the whole "italics for emphasis" use case. I see your point. I like your definition list approach suggested.